.github/workflows/terraform_checks.yml

```diff
@@ -26,27 +26,27 @@ jobs:
 
   check-terraform-validity:
     runs-on: ubuntu-latest
+    env:
+      TERRAFORM_DIRS: |
```

Comment:
We could also make this a list of envs and hardcode `persistent` in the step itself

Reply:
We could, though then we have to call out `global` separately. I think at that point we're just rearranging DRY deck chairs, so I would probably leave this as is unless you feel strongly about it.

Reply:
Ah right I missed `global` - I've been bit before by this very same pattern actually. Agreed we can leave it as is.